Tensile/KernelWriterAssembly.py

```diff
@@ -5140,8 +5140,24 @@ def graAddresses(self, kernel, tP, isPap=False):
       #kStr += self.bomb(0x13) # after addresses and SRD set
     else:
       tmp = self.vgprPool.checkOut(2, "tmp", self.preventVgprOverflowDuringNewTile)
-      kStr += inst("v_mov_b32", vgpr(tmp+0), sgpr("Address%s+0"%tP["tensorChar"]), "" )
-      kStr += inst("v_mov_b32", vgpr(tmp+1), sgpr("Address%s+1"%tP["tensorChar"]), "" )
+            
+      if kernel["StreamK"]:
+        # StreamK partial tile - offset to tile start index
+        tmpOffset = self.sgprPool.checkOut(2, "skStartOffset", preventOverflow=0)
```

Comment:
Probably, we can simply remove tmpOffset and use tmp instead.
Then, we do not need v_mov_b32 to copy from tmpOffset to tmp.

Reply:
sorry. This is sgpr...